src/views/Upload.vue

```diff
@@ -219,61 +233,45 @@ export default {
       this.upload = null
       this.errors = []
 
-      try {
-        const resp = await fetch(`/api/uploads/${this.uploadId}`)
-        const result = (await resp.json()) || { error: (await resp.body) }
-
-        if (resp.ok) {
-          this.upload = result.upload
-        } else {
-          this.alert = {
-            text: `loadUpload Error (${resp.status}): ${result.error}`,
-            level: 'err'
-          }
-        }
-      } catch (e) {
-        this.alert = {
-          text: `loadUpload Unknown Error: ${e.message}`,
+      const result = await getJson(`/api/uploads/${this.uploadId}`)
+      if (result.error) {
+        this.$store.commit('addAlert', {
+          text: `loadUpload Error (${result.status}): ${result.error}`,
           level: 'err'
-        }
+        })
+      } else {
+        this.upload = result.upload
       }
 
       // each time we refresh the upload, also refresh the series
       this.loadSeries()
     },
     loadSeries: async function () {
-      try {
-        const resp = await fetch(`/api/uploads/${this.uploadId}/series`)
-        const result = (await resp.json()) || { error: (await resp.body) }
+      this.series = []
+      this.seriesValid = null
 
-        if (resp.ok) {
-          this.series = result.series
-          this.seriesValid = result.currently_valid
-        } else {
-          this.alert = {
-            text: `loadUpload API Error (${resp.status}): ${result.error}`,
-            level: 'err'
-          }
-        }
-      } catch (e) {
-        this.alert = {
-          text: `loadUpload Unknown Error: ${e.message}`,
+      const result = await getJson(`/api/uploads/${this.uploadId}/series`)
+      if (result.error) {
+        this.$store.commit('addAlert', {
+          text: `loadSeries Error (${result.status}): ${result.error}`,
           level: 'err'
-        }
+        })
+      } else {
+        this.series = result.series
+        this.seriesValid = result.currently_valid
       }
     },
     loadDocuments: async function () {
-      try {
-        const resp = await fetch(`/api/uploads/${this.uploadId}/documents`)
-        const result = (await resp.json()) || { error: (await resp.body) }
+      this.documents = []
```

Comment:
Question: Can we drop this.documents and the corresponding fetch from this component?

I think the extractDocuments code will be useful for building the treasury exports, but it doesn't look like it's rendered in the UI anywhere.

(This can definitely be a separate PR, and I'm happy to take a pass at it.)

Reply:
yeah, i think i left it in from before i refactored this component, but we're not really using it. would be down with this as a separate PR.